tests/test_marquez_wo_client_log.py

```diff
@@ -0,0 +1,170 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging.config
+import os
+import unittest
+import uuid
+
+from marquez_client import Clients
+from marquez_client.models import (DatasetType, JobType)
+from marquez_client.utils import Utils
+
+_NAMESPACE = "my-namespace"
+log = logging.getLogger(__name__)
+
+
+class TestMarquezWriteOnlyClientLog(unittest.TestCase):
+    def setUp(self):
+        log.debug("MarquezWriteOnlyClient.setup(): ")
+        os.environ['MARQUEZ_BACKEND'] = 'log'
+        self.client_wo_log = Clients.new_write_only_client()
+        log.info("created marquez_client_wo_log.")
```

Comment:
If you're testing the log backend, it is best to test the backend contract (put/post) directly rather than through the client. 
Here you are duplicating the client test logic for each backend.

Reply:
we went that way at that time. we can refactor. let's add a new task for it to track it.

Reply:
All you need to test here is that Clients.from_env() returns a LogBackend when os.environ['MARQUEZ_BACKEND'] = 'log'
I would just test that and delete the rest of this test that duplicates existing tests.

Reply:
added test_log_backend.py

Reply:
done.